main.tf

```diff
@@ -1,7 +1,14 @@
+data "google_project" "project" {}
+
+locals {
+  project = var.project != null ? var.project : data.google_project.project.id
```

Comment:
that is the terraform id... please use project_id

Reply:
what is a terraform id?

Reply:
`.id` is terraform internal ID.... `.project_id` is what you need to use..